tests/FoodStorage/test_foodStorage.py

```diff
@@ -22,12 +27,48 @@ def testAddFoods(self):
             self.foodStorage.list()
             )
 
-        # Adding a new food, after listing, should still return a sorted inventory.
-        self.foodStorage.add(food1)
+        # Adding new food, after listing, should still return a sorted inventory.
+        self.foodStorage.addFoods([foodFuture, foodTomorrow, foodToday])
+        self.assertEquals(
+            [(fooddate3, food3), (fooddate1, food1), (fooddate2, food2), 
+            (dateToday, foodToday), (dateTomorrow, foodTomorrow), (dateOutsideRange, foodFuture)], 
+            self.foodStorage.list()
+            )
+
+    def testList(self):
+        self.assertTrue(isEmptyInventory(self.foodStorage))
+
+        self.foodStorage.addFoods(
+            [foodFuture, foodTomorrow, foodNearFuture, foodToday, foodPast, foodYesterday])
+
+        # Grab print output from calling list()
+        capturedListOutput = io.StringIO()
```

Comment:
What is happening here?

Reply:
This is how we grab the result of printing things to standout. I'm basically grabbing the print statement so I can test that it prints as expected below.